app/controllers/api/v1/articles_controller.rb

```diff
@@ -2,8 +2,13 @@ class Api::V1::ArticlesController < ApplicationController
   rescue_from ActiveRecord::RecordNotFound, with: :article_not_found
 
   def index
-    articles = Article.paginate(page: params[:page], per_page: 4)
-    render json: articles, each_serializer: Articles::IndexSerializer, meta: meta_attributes(articles)
+    if params[:category]
+      articles = Article.where(:category => params[:category]).paginate(page: params[:category], per_page: 4)
```

Comment:
use `Article.where(category: params[:category])...` instead to be consistent

Reply:
ok